src/test/mina_automation/docker.ml

```diff
@@ -0,0 +1,130 @@
+open Core
+open Integration_test_lib
```

Comment:
Can new files from `mina_automation` be extracted to a seperate PR, and perhaps a README to be added?

It's hard to understand what's happening here, and I'd prefer to review it separately.